src/um/webauthn.rs

```diff
@@ -0,0 +1,221 @@
+pub type PCWEBAUTHN_USER_ENTITY_INFORMATION = *const WEBAUTHN_USER_ENTITY_INFORMATION;
+pub const WEBAUTHN_CLIENT_DATA_CURRENT_VERSION: DWORD = 1;
+STRUCT!{struct WEBAUTHN_CLIENT_DATA {
+    dwVersion: DWORD,
+    cbClientDataJSON: DWORD,
+    pbClientDataJSON: PBYTE,
+    pwszHashAlgId: LPCWSTR,
+}}
+pub type PWEBAUTHN_CLIENT_DATA = *mut WEBAUTHN_CLIENT_DATA;
+pub type PCWEBAUTHN_CLIENT_DATA = *const WEBAUTHN_CLIENT_DATA;
+pub const WEBAUTHN_CREDENTIAL_TYPE_PUBLIC_KEY: &'static str = "public-key";
+pub const WEBAUTHN_COSE_CREDENTIAL_PARAMETER_CURRENT_VERSION: DWORD = 1;
+STRUCT!{struct WEBAUTHN_COSE_CREDENTIAL_PARAMETER {
+    dwVersion: DWORD,
+    pwszCredentialType: LPCWSTR,
+    lAlg: LONG,
+}}
+pub type PWEBAUTHN_COSE_CREDENTIAL_PARAMETER = *mut WEBAUTHN_COSE_CREDENTIAL_PARAMETER;
+pub type PCWEBAUTHN_COSE_CREDENTIAL_PARAMETER = *const WEBAUTHN_COSE_CREDENTIAL_PARAMETER;
+STRUCT!{struct WEBAUTHN_COSE_CREDENTIAL_PARAMETERS {
+    cCredentialParameters: DWORD,
+    pCredentialParameters: PWEBAUTHN_COSE_CREDENTIAL_PARAMETER,
+}}
+pub type PWEBAUTHN_COSE_CREDENTIAL_PARAMETERS = *mut WEBAUTHN_COSE_CREDENTIAL_PARAMETERS;
+pub type PCWEBAUTHN_COSE_CREDENTIAL_PARAMETERS = *const WEBAUTHN_COSE_CREDENTIAL_PARAMETERS;
+pub const WEBAUTHN_CREDENTIAL_CURRENT_VERSION: DWORD = 1;
+STRUCT!{struct WEBAUTHN_CREDENTIAL {
+    dwVersion: DWORD,
+    cbId: DWORD,
+    pbId: PBYTE,
+    pwszCredentialType: LPCWSTR,
+}}
+pub type PWEBAUTHN_CREDENTIAL = *mut WEBAUTHN_CREDENTIAL;
+pub type PCWEBAUTHN_CREDENTIAL = *const WEBAUTHN_CREDENTIAL;
+STRUCT!{struct WEBAUTHN_CREDENTIALS {
+    cCredentials: DWORD,
+    pCredentials: PWEBAUTHN_CREDENTIAL,
+}}
+pub type PWEBAUTHN_CREDENTIALS = *mut WEBAUTHN_CREDENTIALS;
+pub type PCWEBAUTHN_CREDENTIALS = *const WEBAUTHN_CREDENTIALS;
+pub const WEBAUTHN_CTAP_TRANSPORT_USB: DWORD = 0x01;
+pub const WEBAUTHN_CTAP_TRANSPORT_NFC: DWORD = 0x02;
+pub const WEBAUTHN_CTAP_TRANSPORT_BLE: DWORD = 0x04;
+pub const WEBAUTHN_CTAP_TRANSPORT_TEST: DWORD = 0x08;
+pub const WEBAUTHN_CTAP_TRANSPORT_INTERNAL: DWORD = 0x10;
+pub const WEBAUTHN_CTAP_TRANSPORT_FLAGS_MASK: DWORD = 0x1F;
+pub const WEBAUTHN_CREDENTIAL_EX_CURRENT_VERSION: DWORD = 1;
+STRUCT!{struct WEBAUTHN_CREDENTIAL_EX {
+    dwVersion: DWORD,
+    cbId: DWORD,
+    pbId: PBYTE,
+    pwszCredentialType: LPCWSTR,
+    dwTransports: DWORD,
+}}
+pub type PWEBAUTHN_CREDENTIAL_EX = *mut WEBAUTHN_CREDENTIAL_EX;
+pub type PCWEBAUTHN_CREDENTIAL_EX = *const WEBAUTHN_CREDENTIAL_EX;
+STRUCT!{struct WEBAUTHN_EXTENSION {
+    pwszExtensionIdentifier: LPCWSTR,
+    cbExtension: DWORD,
+    pvExtension: PVOID,
+}}
+pub type PWEBAUTHN_EXTENSION = *mut WEBAUTHN_EXTENSION;
+pub type PCWEBAUTHN_EXTENSION = *const WEBAUTHN_EXTENSION;
+STRUCT!{struct WEBAUTHN_EXTENSIONS {
+    cExtensions: DWORD,
+    pExtensions: PWEBAUTHN_EXTENSION,
+}}
+pub type PWEBAUTHN_EXTENSIONS = *mut WEBAUTHN_EXTENSIONS;
+pub type PCWEBAUTHN_EXTENSIONS = *const WEBAUTHN_EXTENSIONS;
```

Comment:
Nit:  `[P[C]]WEBAUTHN_EXTENSIONS` placement doesn't match header order

Reply:
Should be fixed in 11662566dd9a2305be58503e2927cce8dbf42636.